test/230_dns_unqualified_test.sh

```diff
@@ -26,4 +26,8 @@ check c2
 check c3
 check "$container"
 
+# check that unqualified names are automatically qualified when looking up
+weave_on $HOST1 dns-add $C2 c2 -h name1.$DOMAIN
+assert_dns_a_record $HOST1 c2 name1 $C2 name1.$DOMAIN
+
```

Comment:
You are right: the intention was to test it with a broken resolver. I'll fix it.